data/fro/Leonardi_2003_MortArtu/stemma.gv

```diff
@@ -0,0 +1,43 @@
+digraph {
+Omega[label="Ω", color="grey"]
+X[color="grey"]
+alpha[label="α", color="grey"]
+beta[label="β", color="grey"]
+gamma[label="γ", color="grey"]
+delta[label="δ", color="grey"]
+epsilon[label="ε", color="grey"]
+zeta[label="ζ", color="grey"]
+1[label="", color="grey"]
+2[label="", color="grey"]
+Uetal[label="U et al."]
+Ketal[label="U et al."]
+S1etal[label="S1 et al."]
+
+Omega->X;
+X->alpha;
+alpha->A;
+A->W[style="dashed"];
+X->X1;
+X1->X2;
+X2->beta;
+beta->D;
+X2->gamma;
+gamma->1;
+1->O;
+1->Ac;
+gamma->2[style="dashed";]
+X1->delta;
+delta->2[style="dashed"];
+2->X3;
```

Comment:
It is a tricky one. X3 is a contaminated subarchetype, with ancestors from two distinct families. If we put everything dashed, it would have no «normal» ancestor, but if we don't, we distinguish ourselves from the apparent stemma. Should we have:

```gv
gamma->2[style="dashed";]
delta->2[style="dashed"];
2->X3[style="dashed"];
```

or 

```gv
gamma->2;
delta->2;
2->X3;
```

or even


```gv
gamma->X3;
delta->X3;
```
?